### PR TITLE
Refractoring from Robins scripts

### PR DESCRIPTION
**Pull request by user1b2h5tfaj2**
## Ompalompa refractoring.

flag (1)

### CODE REVIEW

**Review by Linus-Jonsson — APPROVED** [review]
Wonderful!

File: FranticFour/Assets/01_Scripts/Controlls/AssignedController.cs [hunk 1]
```diff
@@ -15,16 +17,17 @@ public int PlayerID
...
     {
         get => usesMouse;
         private set => usesMouse = value;
     }
-    
 
-    [Header("Assigned controls")]
-    [SerializeField] private string vertical;
+    //Control buttons
+    [Header("Assigned controls")] [SerializeField]
```
Comment (NicklasHidesjo) [inline FranticFour/Assets/01_Scripts/Controlls/AssignedController.cs:28]: move [SerializeField] down one line to the same line as the variable like the rest of the script? 👍

File: FranticFour/Assets/01_Scripts/AssignPlayers.cs [hunk 2]
```diff
@@ -1,8 +1,10 @@
 ﻿using UnityEngine;
 
+//Assigns the player from playerSelect
 public class AssignPlayers : MonoBehaviour
 {
-    [SerializeField] private AssignedController[] players = new AssignedController[4];
+    [SerializeField]
```
Comment (NicklasHidesjo) [inline FranticFour/Assets/01_Scripts/AssignPlayers.cs:6]: Move [SerializeField] to the same line as the private AssignedController variable? 👍

**Review by NicklasHidesjo — APPROVED** [review]

File: FranticFour/Assets/01_Scripts/Controlls/AssignedController.cs (hunk 1 above)
Comment (user1b2h5tfaj2) [reply FranticFour/Assets/01_Scripts/Controlls/AssignedController.cs:28]: Fixing in next push :*

File: FranticFour/Assets/01_Scripts/AssignPlayers.cs (hunk 2 above)
Comment (user1b2h5tfaj2) [reply FranticFour/Assets/01_Scripts/AssignPlayers.cs:6]: Fixing in next push :*

**Pull request merged**